bin.js

```diff
@@ -50,14 +50,24 @@ function degit(dir) {
 
 async function patch(dir) {
   try {
-    if (!fs.existsSync(join(dir, 'src/server.js'))) {
+    const serverFile = fs.existsSync(join(dir, 'src/server.js'))
+      ? 'src/server.js'
```

Comment:
Maybe something like:

```js
const serverFile = ['js', 'ts'].find(ext => fs.existsSync(join(dir, `src/server.${ext}`)))

if (!serverFile) { throw new Error(`Expected src/server.{js|ts} to exist, is this really a Sapper project?`) }
```

might be more concise.

Reply:
Will consider this one next time I touch that code 😄

Reply:
yeah I was feeding back as it was being merged haha!